x-pack/plugins/observability_solution/observability_ai_assistant_app/public/utils/shared_providers.tsx

```diff
@@ -15,6 +16,8 @@ import { ObservabilityAIAssistantAppServiceProvider } from '../context/observabi
 import type { ObservabilityAIAssistantAppService } from '../service/create_app_service';
 import type { ObservabilityAIAssistantAppPluginStartDependencies } from '../types';
 
+const storage = new Storage(localStorage);
```

Comment:
any reason why we initiate this outside of the SharedProviders component?

Reply:
It was done in the same way as in the APM app: https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/apm/public/components/routing/app_root/index.tsx#L48